build_tools/github_actions/fetch_test_configurations.py

```diff
@@ -45,6 +45,22 @@ def _get_script_path(script_name: str) -> str:
         "platform": ["linux", "windows"],
         "total_shards": 4,
     },
+    "hipsolver": {
+        "job_name": "hipsolver",
+        "fetch_artifact_args": "--blas --tests",
+        "timeout_minutes": 5,
+        "test_script": f"python {_get_script_path('test_hipsolver.py')}",
+        "platform": ["linux", "windows"],
+        "total_shards": 1,
+    },
+    "rocsolver": {
+        "job_name": "rocsolver",
+        "fetch_artifact_args": "--blas --tests",
+        "timeout_minutes": 120,
+        "test_script": f"python {_get_script_path('test_rocsolver.py')}",
+        "platform": ["linux"],
+        "total_shards": 4,
```

Comment:
The newly added rocsolver tests (I think this was hipsolver only when I last revied) still use 4 shards and have a 120 minute timeout. Is this needed? Otherwise please set lower values as for hipsolver. If a workflow hangs, it will otherwise block resources for 2 hours!